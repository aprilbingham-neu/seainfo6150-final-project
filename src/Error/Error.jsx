import React from "react";
import styles from "./Error.module.css";

const Error = () => {
  return (
    <div className={styles.container}>
      <img
        className={styles.errorImg}
        src="https://wpklik.com/wp-content/uploads/2019/03/A-404-Page-Best-Practices-and-Design-Inspiration.jpg"
        alt="404 error"
      ></img>
      <p className={styles.lostText}>You seem lost.</p>
      <p>The page you are trying to reach doesn't exist.</p>
    </div>
  );
};

export default Error;
