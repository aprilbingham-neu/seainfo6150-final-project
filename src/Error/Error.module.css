.errorImg {
  width: 70%;
  margin: auto;
  left: 0;
  right: 0;
}

.container {
  text-align: center;
  margin: 30px;
}

.lostText {
  font-size: 30px;
  font-family: Verdana, Geneva, Tahoma, sans-serif;
  color: skyblue;
}
