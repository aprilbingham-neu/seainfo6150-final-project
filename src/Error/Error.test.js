import React from "react";
import Error from "./Error";

it("renders the home component correctly", () => {
  const { container } = render(<Error/>);
  expect(container).toMatchSnapshot();
});
