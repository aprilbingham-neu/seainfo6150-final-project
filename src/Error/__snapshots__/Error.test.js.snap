// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the home component correctly 1`] = `
<div>
  <div
    class="container"
  >
    <img
      alt="404 error"
      class="errorImg"
      src="https://wpklik.com/wp-content/uploads/2019/03/A-404-Page-Best-Practices-and-Design-Inspiration.jpg"
    />
    <p
      class="lostText"
    >
      You seem lost.
    </p>
    <p>
      The page you are trying to reach doesn't exist.
    </p>
  </div>
</div>
`;
